addons/microsoft_calendar/tests/test_create_events.py

```diff
@@ -573,7 +573,7 @@ def test_no_videocall_hr_holidays(self, mock_insert, mock_get_events):
             self.skipTest("The 'hr_holidays' module must be installed to run this test.")
 
         self.user_hrmanager = mail_new_test_user(self.env, login='bastien', groups='base.group_user,hr_holidays.group_hr_holidays_manager')
-        self.user_employee = mail_new_test_user(self.env, login='enguerran', password='enguerran', groups='base.group_user')
+        self.user_employee = mail_new_test_user(self.env, login='enguerran', password='enguerran', groups='base.group_user,hr_holidays.group_hr_holidays_employee')
```

Comment:
``microsoft_calendar`` doesn't depend on ``hr_holidays``. Hence this would crash if the latter module is not installed.